pkg/sidecar/config/config.go

```diff
@@ -229,38 +221,22 @@ func (s *ScyllaConfig) setupEntrypoint(ctx context.Context) (*exec.Cmd, error) {
 		}
 	}
 
-	// See if we need to use cpu-pinning
-	// TODO: Add more checks to make sure this is valid.
-	// eg. parse the cpuset and check the number of cpus is the same as cpu limits
-	// Now we rely completely on the user to have the cpu policy correctly
-	// configured in the kubelet, otherwise scylla will crash.
-	if cluster.Spec.CpuSet {
-		cpusAllowed, err := getCPUsAllowedList("/proc/1/status")
-		if err != nil {
-			return nil, errors.WithStack(err)
-		}
-		if err := s.validateCpuSet(ctx, cpusAllowed, s.cpuCount); err != nil {
-			return nil, errors.WithStack(err)
-		}
-		args["cpuset"] = &cpusAllowed
+	cpusAllowed, err := getCPUsAllowedList("/proc/1/status")
+	if err != nil {
+		return nil, errors.WithStack(err)
 	}
+	if err := s.validateCpuSet(ctx, cpusAllowed, s.cpuCount); err != nil {
+		return nil, errors.WithStack(err)
+	}
+	args["cpuset"] = &cpusAllowed
 
-	version := semver.NewScyllaVersion(cluster.Spec.Version)
-
-	klog.InfoS("Scylla version detected", "version", version)
-
-	if len(cluster.Spec.ScyllaArgs) > 0 {
-		if !version.SupportFeatureUnsafe(semver.ScyllaVersionThatSupportsArgs) {
-			klog.InfoS("This scylla version does not support ScyllaArgs. ScyllaArgs is ignored", "version", cluster.Spec.Version)
-		} else {
-			appendScyllaArguments(ctx, s, cluster.Spec.ScyllaArgs, args)
-		}
+	if len(m.AdditionalScyllaDBArguments) > 0 {
+		appendScyllaArguments(ctx, s, strings.Join(m.AdditionalScyllaDBArguments, " "), args)
 	}
 
-	if _, err := os.Stat(scyllaIOPropertiesPath); err == nil && version.SupportFeatureSafe(semver.ScyllaVersionThatSupportsDisablingIOTuning) {
```

Comment:
var `semver.ScyllaVersionThatSupportsDisablingIOTuning` will no longer be used so it can be removed from semver package

Reply:
done